pkg/apis/networking/v1/defaults.go

```diff
@@ -33,3 +33,15 @@ func SetDefaults_NetworkPolicyPort(obj *networkingv1.NetworkPolicyPort) {
 		obj.Protocol = &proto
 	}
 }
+
+func SetDefaults_NetworkPolicySpec(obj *networkingv1.NetworkPolicySpec) {
+	if obj.PolicyType == nil {
```

Comment:
I _think_ we need to check `obj.PolicyType == nil || len(obj.PolicyType == 0)`, right?  Or maybe it's better we just let that case fail validation (I'm thinking someone puts `policyTypes: []` in their yaml).

Can't remember if it's possible to get an empty list at this stage.

Reply:
Pretty sure it's not (possible to get an empty list at this stage).